### PR TITLE
Add hacks for BHNT97

### PR DESCRIPTION
**Pull request by ligi**
PTAL: @MichaelKreil @hannob @ctrlw @strathausen @soulim

### CODE REVIEW

File: _posts/2024-07-30-no97-crowdstruck.md [hunk 1]
```diff
@@ -8,5 +8,32 @@ location: '[c-base](https://www.c-base.org)'
...
+    - https://glitch.com/~delicate-luck-pea
+    - https://github.com/ctrlw/osm4vr
+    - virtual reality, OSM, WebVR, A-Frame
+5. [trees](https://colab.research.google.com/drive/16ZW12CxLJAGTyPmgQ026zqrp-YbslQHV)
+    - by fred
+    - hammock, trees, Berlin, geo data, Python
+6. [Tiny Tiles Designer](https://github.com/strathausen/tinytiles)
+    - by [strathausen](https://github.com/strathausen)
+    - tiles, layout, TypeScript, React
+7. [Alex-o-Meter](gist.github.com/soulim/daa27ea52ffc2ae3f9eb19d47aa0bdf0) - **Hack of the Month**
```
Comment (soulim) [inline _posts/2024-07-30-no97-crowdstruck.md:37]:
```suggestion
7. [Alex-o-Meter](https://gist.github.com/soulim/daa27ea52ffc2ae3f9eb19d47aa0bdf0)
```

File: _posts/2024-07-30-no97-crowdstruck.md [hunk 2]
```diff
@@ -8,5 +8,32 @@ location: '[c-base](https://www.c-base.org)'
...
+    - 3D printing, food
+3. [WEKILL//LEAKS](https://dissent-and-datalove.institute/artworks/wkl/) - a game of courageous publishing and imperialist sh1tf#ckery
+    - by [Dissent and Datalove Institute](https://dissent-and-datalove.institute/)
+    - card game, story telling
+4. [osm4vr](https://delicate-luck-pea.glitch.me)
+    - by [Markus](https://github.com/ctrlw)
+    - https://glitch.com/~delicate-luck-pea
+    - https://github.com/ctrlw/osm4vr
+    - virtual reality, OSM, WebVR, A-Frame
+5. [trees](https://colab.research.google.com/drive/16ZW12CxLJAGTyPmgQ026zqrp-YbslQHV)
```
Comment (soulim) [inline _posts/2024-07-30-no97-crowdstruck.md:31]:
```suggestion
5. [trees](https://colab.research.google.com/drive/16ZW12CxLJAGTyPmgQ026zqrp-YbslQHV) - **Hack of the Month**
```

**Review by soulim — APPROVED** [review]
@ligi thank you for putting it all together! I've added a couple of suggestions:

- Fixed the link on source code of Alex-o-Meter
- Fixed the Hack of Month label as this time my project haven't earned the honor to have it 😉 

Have a great day!

**Comment by strathausen** [comment]
Looks good to me! Thanks!

**Pull request merged**